tests/kafkatest/services/verifiable_producer.py

```diff
@@ -55,16 +79,32 @@ def _worker(self, idx, node):
                     elif data["name"] == "producer_send_success":
                         self.acked_values.append(int(data["value"]))
 
-    @property
-    def start_cmd(self):
-        cmd = "/opt/kafka/bin/kafka-verifiable-producer.sh" \
+    def _kafka_dir(self, node):
+        if node.version == KafkaVersion.TRUNK:
+            return "/opt/kafka"
+        else:
+            return "/opt/kafka-" + node.version
+
+    def start_cmd(self, node):
+
+        cmd = ""
+        if node.version == KafkaVersion.V_0_8_2_1:
+            cmd += "for file in /opt/kafka/tools/build/libs/kafka-tools*.jar; do CLASSPATH=$CLASSPATH:$file; done; "
```

Comment:
Making this conditional on 0.8.2.1 is a bit iffy I agree. 

Mixing jars here is intentional, since versions prior to 0.9.0 don't have VerifiableProducer